tests/ops/integration_tests/saas/test_shipstation_task.py

```diff
@@ -15,7 +16,87 @@ async def test_access_request(
         policy: Policy,
         shipstation_identity_email: str,
     ):
-        await shipstation_runner.access_request(
+
+        key = shipstation_runner.dataset_config.fides_key
+        external_id = shipstation_runner.external_references["customer_id"]
+        access_request = await shipstation_runner.access_request(
             access_policy=policy,
             identities={"email": shipstation_identity_email},
         )
+
+        for customer in access_request[f"{key}:customer"]:
+            assert customer["customerId"] == int(external_id)
+
+        for order in access_request[f"{key}:orders"]:
+            assert order["customerId"] == int(external_id)
+
+        assert_rows_match(
+            access_request[f"{key}:customer"],
+            min_size=1,
+            keys=[
+                "customerId",
+                "createDate",
+                "modifyDate",
+                "name",
+                "company",
+                "street1",
+                "street2",
+                "city",
+                "state",
+                "postalCode",
+                "countryCode",
+                "phone",
+                "email",
+                "addressVerified",
+                "marketplaceUsernames",
+                "tags",
+            ],
+        )
+        assert_rows_match(
+            access_request[f"{key}:orders"],
+            min_size=1,
+            keys=[
+                "orderId",
+                "orderNumber",
+                "orderKey",
+                "orderDate",
+                "createDate",
+                "modifyDate",
+                "paymentDate",
+                "shipByDate",
+                "orderStatus",
+                "customerId",
+                "customerUsername",
+                "customerEmail",
+                "billTo",
+                "shipTo",
+                "items",
+                "orderTotal",
+                "amountPaid",
+                "taxAmount",
+                "shippingAmount",
+                "customerNotes",
+                "internalNotes",
+                "gift",
+                "giftMessage",
+                "paymentMethod",
+                "requestedShippingService",
+                "carrierCode",
+                "serviceCode",
+                "packageCode",
+                "confirmation",
+                "shipDate",
+                "holdUntilDate",
+                "weight",
+                "dimensions",
+                "insuranceOptions",
+                "advancedOptions",
+                "tagIds",
+                "userId",
+                "externallyFulfilled",
+                "externallyFulfilledBy",
+                "externallyFulfilledById",
+                "externallyFulfilledByName",
+                "labelMessages",
+            ],
+        )
```

Comment:
These checks are done automatically as part of the `ConnectorRunner`'s `access_request` so we can remove them